pkg/csl/download_eu.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package csl
+
+import (
+	"fmt"
+
+	"github.com/moov-io/base/log"
+	"github.com/moov-io/watchman/pkg/download"
+)
+
+const uri = "https://webgate.ec.europa.eu/fsd/fsf/public/files/csvFullSanctionsList_1_1/content?token=dG9rZW4tMjAxNw"
```

Comment:
What's this `token` from? I assume we can't use the same token for every user of Watchman.

Reply:
That uri is pulled from the direct access link from the European Union website: https://data.europa.eu/data/datasets/consolidated-list-of-persons-groups-and-entities-subject-to-eu-financial-sanctions?locale=en

Reply:
<img width="1125" alt="Screen Shot 2022-11-15 at 9 30 29 AM" src="https://user-images.githubusercontent.com/115033755/201974390-08eb17d0-33de-4d17-95a9-682bcd3b3bc3.png">

Reply:
`strx.Or(..)` comes from moov-io/base's `strx` package.

Reply:
for sure! I also reached out on their contact us page to see if that changes and how often